tests/e2e/e2e.bats

```diff
@@ -148,3 +148,68 @@
     test "$output" = "ok"
   done
 }
+
+@test "bpf metrics map works" {
+  default_interface=$(ip route | grep default | awk '{print $5}')
+  test ! -z $default_interface
+  ip_address=$(ip addr show "$default_interface" | grep "inet " | awk '{print $2}' | cut -d/ -f1)
+  #gather metrics before the ping
+  output_before=$(kubectl \
+      run -i test-bpfmap \
+      --privileged \
+      --overrides='{"kind":"Pod", "apiVersion":"v1", "spec": {"hostNetwork":true}}' \
+      --image gke.gcr.io/debian-base \
+      --restart=Never \
+      --command \
+      -- sh -c "
+  apt-get update > /dev/null
+  apt-get install iputils-ping libcap2 curl -y --allow-change-held-packages > /dev/null
+  get_icmp6_count() {
+    curl --silent localhost:8881/metrics | grep protocol=\\\"ICMPv6\\\" | cut -d ' ' -f2
+  }
+  get_icmp_count() {
+      curl --silent localhost:8881/metrics | grep protocol=\\\"ICMP\\\" | cut -d ' ' -f2
+  }
+  get_icmp6_count
+  get_icmp_count")
+  # perform the ping
+  kubectl \
+        run -i test-bpfmap1 \
+        --privileged \
+        --image gke.gcr.io/debian-base \
+        --restart=Never \
+        --command \
+        -- sh -c "
+    apt-get update > /dev/null
+    apt-get install iputils-ping libcap2 curl -y --allow-change-held-packages > /dev/null
+    ping -c 7 64:ff9b::$ip_address
+  "
```

Comment:
see the modified testin #40, use the image `registry.k8s.io/e2e-test-images/agnhost:2.39 ` that already has curl and ping and does not need to install them